packit_service/worker/jobs.py

```diff
@@ -471,6 +471,27 @@ def is_project_public_or_enabled_private(self) -> bool:
 
         return True
 
+    def allow_non_matching_events(self):
+        """Do explicit exceptions for non matching event/jobs triggers
+
+        Returns:
+            list of jobs
+        """
```

Comment:
I am still wondering why is this needed and if we can't fix this elsewhere. This is because of the `commit` vs `pull_request` trigger, isn't it?

TBH, I was a bit confused by the method name and docstring. Could you please rename it to something like `check_manual_mapping`/`check_explicit_matching`/.. or something like that? (From the name, I thought we are allowing any combination...;)  And then, I've seen the word `exception` and was confused completely..;)

Reply:
Yes this is because of the `commit` vs `pull_request` trigger. 
I may be wrong but I don't see how to allow this without this hack.
But we could probably ask to the users to put in their configuration a job with trigger `pull_request` for the `koji_build` if they want to trigger it through a comment in a `pull_request`.
I copied the config file from osbuilder project and they do not have such a job in their config, they only have a `koji_build` job triggered by a `commit`and I want my code to be able to trigger a build with their configuration.
But I can remove this lines if we agree that a change in the config file should be needed.

Reply:
No, I don't think we should ask users to add another job. (It's same as with the `propose-downstream` where nothing is needed to be able to retrigger the job by an issue comment.)

We can leave it as is and try to refactor it later.